<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="DB">
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Введите путь к файлу с БД</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openDB">
       <property name="text">
        <string>Открыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="WTable" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>441</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="LTable">
      <item>
       <widget class="QLineEdit" name="Table">
        <property name="placeholderText">
         <string>Введите имя таблицы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="showTable">
        <property name="text">
         <string>Показать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>421</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="Redactor" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>441</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="Buttons">
        <item>
         <widget class="QSpinBox" name="spinBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="add">
          <property name="text">
           <string>add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="save">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_2">
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="Fields">
        <item row="0" column="0">
         <widget class="QLineEdit" name="nameField1">
          <property name="text">
           <string>sID</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="field1"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="nameField2">
          <property name="text">
           <string>sName</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="field2"/>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="nameField3">
          <property name="text">
           <string>GPA</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="field3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="LQuery">
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameTable">
        <property name="placeholderText">
         <string>Таблица</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="entFields">
        <property name="placeholderText">
         <string>Поля</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="where">
        <property name="placeholderText">
         <string>Условие</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="queryTable">
        <property name="text">
         <string>Запрос</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Работа с таблицами</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
